src/genomic_tools_lib/file_formats/Parquet.py

```diff
@@ -25,11 +26,12 @@ def _deplete_genotype_variants_to_record_batch(genotype, individual_ids):
 
 #The conversion to record batch is because of a bug in pyarrow for flavor="spark"
 def _to_record_batch(df):
-    data =[]
-    names = list(df.columns.values)
-    for c in names:
-        data.append(pa.array(df[c]))
-    return  pa.RecordBatch.from_arrays(data, names)
+    return pa.RecordBatch.from_pandas(df)
+    # data =[]
```

Comment:
There was a performance reason to load from arrays, but if you are changing to `from_pandas` then I wouldn't keep the old code around in a comment.